lib/cdo/crowdin/utils.rb

```diff
@@ -127,6 +127,9 @@ def download_changed_files
         etags[code].merge! downloaded_files
         File.write @etags_json, JSON.pretty_generate(etags)
       end
+      # Makes sure these files get written even if nothing was downloaded.
```

Comment:
Consider moving this corner case to right after reading from `@files_to_download`. If `files_to_download.keys.length` is 0, write "{}" to `@files_to_sync_out` and exit.

Reply:
Also, don't need to overwrite `@files_to_download`.

Reply:
As an alternative, I changed to the code so it initializes `@files_to_sync_out` at the beginning of the function instead. Does that look good to you?